index.js

```diff
@@ -1,81 +1,141 @@
 const fs = require('node:fs');
 const path = require('node:path');
 const { Client, Collection, GatewayIntentBits } = require('discord.js');
-require("dotenv").config();
-const { Player } = require("discord-player");
-const { YoutubeiExtractor } = require("discord-player-youtubei");
-const { ZiExtractor } = require("ziextractor");
+require('dotenv').config();
+const { Player } = require('discord-player');
+const { YoutubeiExtractor } = require('discord-player-youtubei');
+const { ZiExtractor } = require('ziextractor');
+const chalk = require('chalk');
+const { table } = require('table');
+
 const client = new Client({
-    intents: [
-        GatewayIntentBits.Guilds,
-        GatewayIntentBits.GuildVoiceStates
-    ]
+  intents: [GatewayIntentBits.Guilds, GatewayIntentBits.GuildVoiceStates],
 });
 
 const player = new Player(client, {
-    skipFFmpeg: false
+  skipFFmpeg: false,
 });
 player.setMaxListeners(100);
 player.extractors.register(YoutubeiExtractor, {
-    authentication: process.env?.YoutubeAUH || "",
-    streamOptions: {
-        useClient: "ANDROID"
-    }
+  authentication: process.env?.YoutubeAUH || '',
+  streamOptions: {
+    useClient: 'ANDROID',
+  },
 });
 player.extractors.register(ZiExtractor, {});
-player.extractors.loadDefault((ext) => !["YouTubeExtractor"].includes(ext));
+player.extractors.loadDefault(ext => !['YouTubeExtractor'].includes(ext));
 
 // player.on("debug", console.log)
 
 client.commands = new Collection();
 client.functions = new Collection();
 client.cooldowns = new Collection();
 
-const loadFilesAsync = async (directory, collection) => {
+const loadFiles = (directory, collection) => {
+  try {
     const folders = fs.readdirSync(directory);
-    console.log(`========== Load ${path.basename(directory)} ==========`);
-    
-    await Promise.all(folders.map(async (folder) => {
-        const folderPath = path.join(directory, folder);
-        const files = fs.readdirSync(folderPath).filter(file => file.endsWith('.js'));
-
-        await Promise.all(files.map(async (file) => {
-            const filePath = path.join(folderPath, file);
-            const module = require(filePath);
-
-            if ('data' in module && 'execute' in module) {
-                console.log(`Loaded ${folder}:`, module.data.name);
-                collection.set(module.data.name, module);
-            } else {
-                console.log(`[WARNING] The ${folder} at ${filePath} is missing a required "data" or "execute" property.`);
-            }
-        }));
-    }));
-};
+    console.log(`========== Load ${directory.split('\\').slice(-1)?.at(0)} ==========`);
+    const clientCommands = [];
+    for (const folder of folders) {
```

Comment:
chỗ này sài Promise.all nhanh hơn for nà

Reply:
mình sửa rồi á 😁